activerecord/lib/active_record/core.rb

```diff
@@ -349,6 +349,26 @@ def init_with(coder)
       self
     end
 
+    ##
+    # Initializer used for instantiating objects that have been read from the
+    # database.  +attributes+ should be an attributes object, and unlike the
```

Comment:
double space after `# database.` redundant?

Reply:
I'm not sure.  We learned double space after punctuation in school because everything was fixed width font at that time.  These days we don't need it because most stuff (except code) isn't fixed width.  Since code is fixed width, I go with double space but I'm not sure what our style guide is in this case.  @fxn?